packages/cli/src/services/user.service.ts

```diff
@@ -41,6 +41,13 @@ export class UserService {
 		return this.userRepository.find(options);
 	}
 
+	async findManybyIds(userIds: string[]) {
+		return this.findMany({
+			where: { id: In(userIds) },
+			relations: ['globalRole'],
```

Comment:
I'm not sure if this additional `globalRole` data is actually used anywhere in this context.